widgets/google-map/tpl/static-map.php

```diff
@@ -1,3 +1,9 @@
+<?php
+if( !empty( $destination_url ) ) echo '<a href="' . $destination_url . '" ' . ( $new_window ? 'target="_blank"' : '' ) . '>'; ?>
```

Comment:
For this sort of template stuff, keeping HTML as HTML and using alternative control structure makes for neater (and more consistent) template files. Echoing HTML tags shouldn't happen often, especially in template files.